LSP.sublime-settings

```diff
@@ -203,6 +203,13 @@
       }
       ]
     },
+    "Metals":
```

Comment:
There was an unspoken convention of lowercase names here, as it was only for looking up configs/clients case-sensitively by name in code. 
I understand it's shown to users now, but is it ok if we stick with lowercase? I'd like to avoid users having issues overriding a "Metals" config with a "metals" key in their user config.